localization/english/religion/wc_gods_and_titles_l_english.yml

```diff
@@ -0,0 +1,421 @@
+﻿l_english:
+ ANCESTORS_NAME:0 "ancestors"
+ ANCESTORS_POSSESSIVE:0 "ancestors'"
+ 
+ DISORDER_NAME:0 "the Disorder"
+ DISORDER_POSSESSIVE:0 "the Disorder's"
+ 
+ TEMPLE_NAME:0 "temple"
+ TEMPLE_PLURAL:0 "temples"
+ 
+ EYE_NAME:0 "eye"
+ LIBRAM_NAME:0 "librams"
+ 
+ HIGH_MAGISTER_NAME:0 "High Magister"
+ MAGISTERIA_NAME:0 "Magisteria"
```

Comment:
Wouldn't "Magistra" be better for female version? @DukeJeans @LordSilvermane ?

Reply:
> Wouldn't "Magistra" be better for female version? @DukeJeans @LordSilvermane ?

That's not a female version, it's the name of a title.

Reply:
Oh, my bad, sry.